imagetest/test_suites/windows/gvnic_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build cit
```

Comment:
i don't think we need any test in the windows suite, do we? can we make the gvnic_test in the network suite multi platform?